app/controllers/api/v1/study_files_controller.rb

```diff
@@ -478,7 +479,9 @@ def parse
               @study_file.update(parse_status: 'parsing')
               genes.update(parse_status: 'parsing')
               barcodes.update(parse_status: 'parsing')
```

Comment:
We don't want to move mtx file parsing over to ingest pipeline yet--the code isn't ready

Reply:
Ah, good catch.  I thought I had undone that.